dispatcher/main.go

```diff
@@ -201,6 +201,27 @@ func getVolumes(run *models.Run) (volumes []corev1.Volume) {
 			},
 		})
 
+	if len(droidMetadata.SecretFiles) == 0 {
+		return
+	}
+
+	for _, file := range droidMetadata.SecretFiles {
+		volumes = append(volumes,
+			corev1.Volume{
+				Name: common.StorageVolumeNameSecrets,
+				VolumeSource: corev1.VolumeSource{
+					Secret: &corev1.SecretVolumeSource{
+						SecretName: run.GetSecretName(droidMetadata),
+						Items: []corev1.KeyToPath{
+							{
+								Key:  file.Value,
+								Path: file.Path,
+							},
+						},
+					},
+				},
+			})
+	}
 	return
```

Comment:
Do we need this?

Reply:
Yes, documentation [here](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod)

Reply:
no. I meant do we need the additional `return` since the function comes to a natural end here.

Reply:
We need it, yes

Reply:
OK!